src/ugeneui/src/project_support/ProjectLoaderImpl.cpp

```diff
@@ -814,45 +815,36 @@ ProjectDialogController::ProjectDialogController(ProjectDialogController::Mode m
     createButton = buttonBox->button(QDialogButtonBox::Ok);
     setModal(true);
     fileEditIsEmpty = false;
-    QString lastDir = AppContext::getSettings()->getValue(SETTINGS_DIR + "last_dir", QString(""), true).toString();
-    projectFolderEdit->setText(lastDir);
 
     if (m == Save_Project) {
         setWindowTitle(ProjectLoaderImpl::tr("Save project as"));
         createButton->setText(ProjectLoaderImpl::tr("Save"));
         projectNameEdit->setText(AppContext::getProject()->getProjectName());
         QString url = AppContext::getProject()->getProjectURL();
         if (!url.isEmpty()) {
-            QFileInfo fi(url);
-            projectFileEdit->setText(fi.completeBaseName());
-            projectFolderEdit->setText(fi.absolutePath());
+            projectFilePathEdit->setText(url);
         } else {
             setupDefaults();
         }
     } else {
         setupDefaults();
     }
     //projectFolderEdit->setReadOnly(true);
-    if (projectFileEdit->text().isEmpty()) {
+    if (projectFilePathEdit->text().isEmpty()) {
         fileEditIsEmpty = true;
     }
-    connect(folderSelectButton, SIGNAL(clicked()), SLOT(sl_folderSelectClicked()));
-    connect(projectFileEdit, SIGNAL(textEdited(const QString &)), SLOT(sl_fileNameEdited(const QString &)));
+    connect(fileSelectButton, SIGNAL(clicked()), SLOT(sl_fileSelectClicked()));
+    connect(projectFilePathEdit, SIGNAL(textEdited(const QString &)), SLOT(sl_fileNameEdited(const QString &)));
     connect(projectNameEdit, SIGNAL(textEdited(const QString &)), SLOT(sl_projectNameEdited(const QString &)));
```

Comment:
Shouldn' t this line be removed?

Reply:
It's okay. There were 3 of 2 connected QLineEdits, now 2 of 2.